docs/about/_index.html

```diff
@@ -0,0 +1,39 @@
+---
+title: About Grafana-Operator
+linkTitle: About
+menu:
+  main:
+    weight: 10
+---
+
+{{< blocks/section >}}
+
+<div class="col-12">
+<h1 class="text-center">About Grafana-Operator</h1>
+
+<h2>Creation</h2>
```

Comment:
This is the feeling I have gotten but I don't know the official version. But I guess I wanted to show a bit where the operator came from and where we are going.